module/x/gravity/keeper/keeper_valset.go

```diff
@@ -256,6 +256,28 @@ func (k Keeper) GetCurrentValset(ctx sdk.Context) (types.Valset, error) {
 	if len(validators) == 0 {
 		return types.Valset{}, types.ErrNoValidators
 	}
+
+	if k.shutdown {
+		za := types.ZeroAddress()
+		bridgeValidators := []*types.InternalBridgeValidator{
+			{
+				Power:           normalizeValidatorPower(1, sdk.NewInt(1)),
+				EthereumAddress: za,
```

Comment:
this have to be tested in the smart contract. We need to check if there are no sanity checks in Solidity.
Could you confirm (and maybe @EbrahimUmee  as well)?

Reply:
I've checked but it would be good to have more eyes on it. I've tested it and it seems to work fine: https://sepolia.etherscan.io/tx/0x271b32d9d5fe204bc2a0cd0cbc2661d884d63e04bc7ef5a5843e4537808db481#eventlog